common/isolationgroup/isolationgroupapi/domain-api-handlers.go

```diff
@@ -28,11 +28,26 @@ import (
 	"github.com/uber/cadence/common/types"
 )
 
-func (handlerImpl) GetDomainState(ctx context.Context, request types.GetDomainIsolationGroupsRequest) (*types.GetDomainIsolationGroupsResponse, error) {
-	panic("not implemented")
+func (z *handlerImpl) GetDomainState(ctx context.Context, request types.GetDomainIsolationGroupsRequest) (*types.GetDomainIsolationGroupsResponse, error) {
+	res, err := z.domainHandler.DescribeDomain(ctx, &types.DescribeDomainRequest{
```

Comment:
we are not using domain cache?

Reply:
Not for the API, a cache serves no purpose here